resources/views/students/create.blade.php

```diff
@@ -75,7 +74,7 @@
             <div class="field-body">
                 <div class="field">
                     <div class="control is-expanded">
-                        <input class="input" type="text" id="BirthDate" name="BirthDate" placeholder="Birth Date" value="{{\Carbon\Carbon::parse($student->Birthday)->format('m/d/Y')}}" required/>
+                        <input class="input" type="text" id="Birthday" name="Birthday" placeholder="Date of Birth" value="{{\Carbon\Carbon::parse($student->Birthday)->format('m/d/Y')}}" required/>
```

Comment:
If possible, try to process the data in the controller and pass a formatted value to the view.